modules/apps/portal-workflow/source-formatter-suppressions.xml

```diff
@@ -2,6 +2,7 @@
 
 <suppressions>
 	<source-check>
+		<suppress checks="IllegalTaglibsCheck" files="portal-workflow-task-web/src/main/resources/META-INF/resources/edit_workflow_task\.jsp" />
```

Comment:
@rijema could you please ping me on Slack about this? I'd like to know if there is another way to avoid this suppression

Reply:
Not need